src/os.cc

```diff
@@ -57,6 +38,29 @@
 #  endif
 #endif
 
+#if FMT_USE_FCNTL
+#  ifdef _WIN32
```

Comment:
We can merge these two now to reduce nesting:

```
#if FMT_USE_FCNTL && defined(_WIN32)
```

Reply:
Changed to `#if defined(FMT_USE_FCNTL) && defined(_WIN32)` (see below)